### DISCUSSION TITLE
What's the equivalent of `require'mason-lspconfig'.setup()` for linters?

### DISCUSSION THREAD

**Discussion by gzagatti** [mason-org/mason.nvim#1719] [Q&A]
I was installing linters with Mason, but I did not find in the documentation a way to programattically specify them in `init.lua`. 

If I add linters to  `require'mason-lspconfig'.setup(ensure_installed = {})`, I get a message saying that the LSP package is not available.

```
[mason-lspconfig.nvim] Server "mypy" is not a valid entry in ensure_installed. Make sure to only provide lspconfig server names.
```

Is it possible to do that? Am I missing something?

**Comment by eleloi** [comment]
Just meeted this issue today... any news?
